system-addon/content-src/message-center/components/SnippetBase.jsx

```diff
@@ -6,11 +6,14 @@ const defaultStyles = {
     bottom: 0,
     left: 0,
     right: 0,
-    padding: "20px",
-    backgroundColor: "white",
+    padding: "12px",
+    backgroundColor: "var(--newtab-card-background-color)",
```

Comment:
@rlr do you think this is a good option for the snippets background so it works with dark theme?

Reply:
👍 as long as you are OK with the color being coupled to the card color.